projects/netgrif-components/src/lib/data-fields/i18n-field/i18n-text-field/i18n-text-field.component.spec.ts

```diff
@@ -12,35 +12,40 @@ import {
     TestConfigurationService,
     TranslateLibModule,
     UserResourceService,
-    WrappedBoolean
+    WrappedBoolean,
+    LanguageService
 } from '@netgrif/components-core';
 import {HttpClientTestingModule} from '@angular/common/http/testing';
-import {Component} from '@angular/core';
+import {Component, CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
 import {FormControl} from '@angular/forms';
+import {AngularResizedEventModule} from 'angular-resize-event';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
 
 describe('I18nTextFieldComponent', () => {
     let component: I18nTextFieldComponent;
     let fixture: ComponentFixture<TestWrapperComponent>;
+    let service: LanguageService;
 
     beforeEach(async () => {
         await TestBed.configureTestingModule({
             imports: [
                 MaterialModule,
-                HttpClientTestingModule,
-                TranslateLibModule
+                AngularResizedEventModule,
+                BrowserAnimationsModule,
+                TranslateLibModule,
+                HttpClientTestingModule
             ],
             providers: [
                 {provide: AuthenticationMethodService, useClass: MockAuthenticationMethodService},
+                {provide: ConfigurationService, useClass: TestConfigurationService},
                 {provide: AuthenticationService, useClass: MockAuthenticationService},
-                {provide: UserResourceService, useClass: MockUserResourceService},
-                {provide: ConfigurationService, useClass: TestConfigurationService}
+                {provide: UserResourceService, useClass: MockUserResourceService}
             ],
-            declarations: [I18nTextFieldComponent, TestWrapperComponent]
-        })
-            .compileComponents();
-    });
-
-    beforeEach(() => {
+            declarations: [I18nTextFieldComponent, TestWrapperComponent],
+            schemas: [CUSTOM_ELEMENTS_SCHEMA]
```

Comment:
why?

Reply:
CUSTOM_ELEMENT_SCHEMA is used in each dataField test so I included it in these two tests too.
Maybe @Kovy95 knows more about it.

Reply:
i think its okay